contracts/SponsorFaucet.sol

```diff
@@ -36,7 +37,6 @@ contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard {
     /*** Dapp dev calls ***/ 
     //apply cfx for gas & storage
     function applyFor(address dapp) public nonReentrant whenNotPaused {
```

Comment:
`todo` is removed, but I cannot see any code to check sponsor value before set new sponsor.

In details, if current sponsor value is larger, the tx call will fail. Because, anyone could sponsor a contract.

Reply:
fixed